builtin/providers/aws/provider.go

```diff
@@ -215,6 +215,9 @@ func Provider() terraform.ResourceProvider {
 			"aws_lambda_permission":                        resourceAwsLambdaPermission(),
 			"aws_launch_configuration":                     resourceAwsLaunchConfiguration(),
 			"aws_lb_cookie_stickiness_policy":              resourceAwsLBCookieStickinessPolicy(),
+			"aws_elb_load_balancer_policy":                 resourceAwsLoadBalancerPolicy(),
```

Comment:
I personally think these resources should be renamed

we use `aws_lb_cookie_stickiness_policy` as an ELB addition, I think we should do the same thing for `aws_lb_policy`, `aws_backend_server_policy` and `aws_lb_listener_policy`